backend/Controller/Comparers/Food/FoodOldFirst.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace backend.Controller.Comparers.Food
+{
+    public class FoodOldFirst : IComparer<Entities.Food>
+    {
+        public FoodOldFirst()
+        { }
```

Comment:
see `FoodExpensiveFirst.cs` line 10, there is no space between `{` and `}` there. We should be consistent with our formatting. I'd say leave it as `{ }`